packages/sitecore-jss-nextjs/src/middleware/personalize-middleware.ts

```diff
@@ -234,6 +261,8 @@ export class PersonalizeMiddleware {
     // Disable preflight caching to force revalidation on client-side navigation (personalization may be influenced)
     // See https://github.com/vercel/next.js/issues/32727
     response.headers.set('x-middleware-cache', 'no-cache');
+    // Share rewrite path with following executed middlewares
+    response.headers.set('x-sc-rewrite', rewritePath);
```

Comment:
Wonder if we should also be setting the 'sc_site' cookie in the personalize and redirects middleware in the case where it wasn't present